src/app/(pages)/fuer-eltern/schliessfaecher/page.tsx

```diff
@@ -0,0 +1,36 @@
+export default function Page() {
+    return(
+    <div>
+        <h1>Schließfächer am MCG</h1>
+        <section>
+            <div className="tutorial">
+                <h2>Wie miete ich ein Schließfach?</h2>
+                <p>
+                    Schließfächer können von Schülerinnen und Schülern gemietet werden. Die Miete beträgt 2,60 € pro Monat (31,20 €/ 12 Monate) und mit <a href="https://www.astradirect.de/schueler">Schutzbrief</a> 43,20 € pro 12 Monate.
+                    <br></br>
+                    Die Schließfächer werden vom Unternehmen ,,Astradirect" zur Verfügung gestellt.
+                </p>
+                <p>
+                    <a href="https://www.astradirect.de/schliessfaecher/" target="_blank">Hier</a> können Sie sich über die Schließfächer informieren.
+                </p>
+                <br></br>
+                <h2>Wie miete ich ein Schließfach?</h2>

```

Comment:
This heading is identical to the one on line 7. I would change this one to "Anleitung".

Reply:
Done.